### ISSUE TITLE
FEATURE - What's New Screen

### ISSUE THREAD

**Issue by mikaelacaron** [mikaelacaron/brain-marks#163]
## Describe the feature you want the app to have that it currently doesn't
* Add a new section to the settings tab (before the appearance section) called "Updates" and the row is called "what's new"
* Show markdown file of the updates for each version
```markdown
# v1.2
* list of updates
* list of stuff

# v1.2 
* list of updates
* list of stuff
```

## Describe alternatives you've considered
* Possibly fetching JSON that's stored in GitHub? (So that it can be updated in GitHub, and we don't have to push a new App Store update for each? (But this isn't super important, cause a new App Store version will be deployed, and changes to the md file can be made then)

## Additional context
* You can grab the the previous version updates from the App Store, and tag me in a comment to ask me about the Hacktoberfest updates (for the new version after Oct 2022)

**Comment by SuzGupta** [comment]
Would love to give this a try.

**Comment by SuzGupta** [comment]
The initial changes to the Settings view are trivial (already done as shown in attached screencap) and I should be able to figure out how to show the new view as well, but I am less clear on how to get the information. Should I use the App Store Connect API as described here: https://www.avanderlee.com/swift/app-store-connect-api-adoption/? 
![SettingsView](https://user-images.githubusercontent.com/5115383/197368724-890ef430-38cf-4beb-ad7e-2629bcc38e27.png)

**Comment by SuzGupta** [comment]
If using this API is the correct approach, a key needs to be generated within App Store Connect. This is explained in the SwiftLee article linked above.

**Comment by mikaelacaron** [comment]
@SuzGupta naaahhh just hard code a description!

**Comment by SuzGupta** [comment]
Whew, that sounds so much easier :D

**Comment by mikaelacaron** [comment]
@SuzGupta  Yes! I don't care at all that it's not autoupdating, considering this app doesn't have constant updates at the moment!

**Comment by le0zh0u** [comment]
Will this support multiple languages?

**Comment by mikaelacaron** [comment]
@le0zh0u that's a good point! I didn't think of how it would work
